stdlib/src/builtin/float_literal.mojo

```diff
@@ -239,7 +239,38 @@ struct FloatLiteral(
             return truncated
         return truncated + 1
 
-    # TODO: implement __round__
+    @always_inline("nodebug")
+    fn __round__(self, ndigits: IntLiteral = 0) -> Self:
```

Comment:
Implemented `Roundable` trait here: https://github.com/msaelices/mojo/commit/26906cafbe88f57087d4dba9736c7a911d13f57a

I'm not sure if adding a `ndigits` function to `Roundable` would break integers, and that parameter on those types has no meaning, right?

Reply:
It has the same as for floating point numbers, it's just that for non-negative `ndigits` values, the `round` function acts like identity on integers:
```python
round(5232323,-2) == 5232300
```
If you are interested, I would love to see a followup patch with improving the `Roundable` trait and the implementation for other types!

Reply:
Sure. I will create a separated PR with the improvement.

Reply:
Amazing, thanks, please ping me when that's done!

Reply:
@laszlokindrat  It's here: https://github.com/modularml/mojo/pull/2656